src/main/java/com/blackducksoftware/integration/hub/alert/accumulator/RealTimeListener.java

```diff
@@ -51,8 +55,12 @@ public RealTimeListener(final Gson gson, final ChannelTemplateManager channelTem
     @JmsListener(destination = RealTimeEvent.TOPIC_NAME)
     @Override
     public void receiveMessage(final String message) {
-        final RealTimeEvent event = getEvent(message);
-        final List<AbstractChannelEvent> events = notificationProcessor.processNotifications(DigestTypeEnum.REAL_TIME, event.getNotificationList());
-        channelTemplateManager.sendEvents(events);
+        try {
+            final RealTimeEvent event = getEvent(message);
+            final List<AbstractChannelEvent> events = notificationProcessor.processNotifications(DigestTypeEnum.REAL_TIME, event.getNotificationList());
+            channelTemplateManager.sendEvents(events);
+        } catch (final Exception e) {
```

Comment:
When does an exception get thrown here?

Reply:
Does it have to do with templating?

Reply:
We use this as a catch all, because if an exception does get thrown the JmsListener will add the message back into the Queue and keep trying to send it